charts/matrixbot/templates/configmap.yaml

```diff
@@ -33,8 +33,8 @@ data:
         mxid: {{ .Values.botUser | quote }}
         password: {{ .Values.botPassword | quote }}
         homeserver: {{ .Values.homeserver }}
-        {{ if eq .Values.encryption.enabled true -}}
```

Comment:
Well... I'd remove the values.yml encryption value if the default is enabled anyway. All the stuff within the if is optional afaik.

Reply:
if you plan to use encryption ( for real ) that values are really important

Reply:
that's why I'd keep the switch

Reply:
instead if you don't care about encryption, the instance will be encryption ready, but you won't be forced to set up these additional fields since are not relevant for your use case

wdyt ?

Reply:
Let's see how it goes.